data/campaigns/The_Rise_Of_Wesnoth/scenarios/11_Southbay_in_Winter.cfg

```diff
@@ -182,15 +182,15 @@
         [/message]
         [message]
             speaker=King Addroran IX
-            message= _ "Hmm. There is wisdom in your words, and my end should be in this place. We can hold out for the rest of the winter here. When the orcs last came it was fall and the harvest was in. We can hunt the great schools of fish that live under the ice. You might even be able to depart in well provisioned ships!"
+            message= _ "Hmm. There is wisdom in your words, and my end should be in this place. We can hold out for the rest of the winter here. When the orcs last came it was fall and the harvest was in. We can hunt the great schools of fish that live under the ice. You might even be able to depart in well-provisioned ships!"
         [/message]
         [message]
             speaker=Lady Jessene
             message= _ "Don’t you intend to come with us?"
         [/message]
         [message]
             speaker=King Addroran IX
-            message= _ "No. Flee, and flee east. It is my duty to hold this city as long as I can for the sake of all who can be evacuated. That means that it shall fall on Haldric and you to lead the evacuation."
+            message= _ "No. Flee, and flee east. It is my duty to hold this city as long as I can for the sake of all who can be evacuated. This means that it shall fall on Haldric and you to lead the evacuation."
```

Comment:
I think "that" is more correct in this line.

If you made the change because "that" appears twice in the same sentence, you could just remove the second instance, though personally I don't think that's an issue.

Unrelated side note: maybe also "upon Haldric"?

Reply:
I followed the suggestion here (celmin). A google search shows me that upon and on are practically interchangeable and don't make much difference in meaning. Keeping the original text.

Reply:
Yeah, "upon" doesn't change the meaning, just gives a different feel to it. More formal maybe? But it's not really an issue.